src/databricks/sql/client.py

```diff
@@ -297,6 +298,58 @@ def _check_not_closed(self):
         if not self.open:
             raise Error("Attempting operation on closed cursor")
 
+    def _handle_staging_operation(self):
+        """Make HTTP request using instructions provided by server"""
+
+        row = self.active_result_set.fetchone()
+
+        # TODO: Handle headers. What format will gateway send? json? plaintext?
+        operation, presigned_url, local_file, headers = (
+            row.operation,
+            row.presignedUrl,
+            row.localFile,
+            None,
+        )
+
+        operation_map = {
+            "PUT": requests.put,
+            "GET": requests.get,
+        }
+
+        if operation not in operation_map:
+            raise Error(
+                "Operation {} is not supported. Supported operations are {}".format(
+                    operation, ",".join(operation_map.keys())
```

Comment:
any reason we are not using f strings?
https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python

sounds a better approach comparing to `.format()` to me.

Reply:
Python actually has three interchangeable string formatting syntaxes. 

1. `f"This is a {value}"`
2. `"This is a {}".format(value)`
3. `"This is a %s" % value`

PySQL uses all three of them in different places. I'll switch these to f-strings if that's what you like :)